Sources/Request.swift

```diff
@@ -4,14 +4,17 @@ public struct Request: Message {
     public var version: Version
     public var headers: Headers
     public var body: Body
-    public var storage: [String: Any] = [:]
+    public var cookies: [String: String]
```

Comment:
@Danappelxx Cookies are `[String: String]` for `Request`. I think this makes good use of Swift's native types and conforms well to the RFC spec. 

The only non-naming issue here is being able to extend a `Cookie` object, but anything you'd want to add to `Cookie` you could add to `Dictionary<String, String>`. 

What are others' thoughts on this?